packages/flutter/lib/src/widgets/inherited_model.dart

```diff
@@ -34,7 +34,7 @@ import 'framework.dart';
 /// ```dart
 /// class MyModel extends InheritedModel<String> {
 ///   // ...
-///   static MyModel of(BuildContext context, String aspect) {
+///   static MyModel? of(BuildContext context, String aspect) {
```

Comment:
Ideally, we should wrap this in a `{@tool snippet}` section, so this code gets analyzed and never goes out of date again. Wanna give that a try in this PR?